zerver/lib/user_groups.py

```diff
@@ -29,6 +29,24 @@ class UserGroupDict(TypedDict):
     can_mention_group_id: int
 
 
+def user_ids_to_users(user_ids: Sequence[int], realm: Realm) -> List[UserProfile]:
```

Comment:
Why is this moved to this file? I think `zerver/lib/users.py` feels much more natural to me given that this function doesn't have any reference to groups.

Reply:
I see this is covered in the commit message; I prefer to leave it in `users.py` because that way it'll be easier to find for other code that does need to do this task; the function itself is perfectly reusable.

Otherwise this PR lgtm.

Reply:
That's fine by me.  I was sort of on the fence.